### PR TITLE
 [PTBER072022] berndHavekost

### PR DESCRIPTION
**Pull request by hvkst**
everything done, added a little responsiveness

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "liveServer.settings.port": 5501
```
Comment (trebeljahr) [inline .vscode/settings.json:2]: this doesn't need to be committed ^^

File: apple-pie/index.html [hunk 2]
```diff
@@ -4,18 +4,79 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Apple Pie Recipe</title>
-    <!-- don't forget to link your styles -->
+    <link rel="stylesheet" href="styles/style.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
```
Comment (trebeljahr) [inline apple-pie/index.html:10]: nice font import :)

File: apple-pie/index.html [hunk 3]
```diff
@@ -4,18 +4,79 @@
...
+            <li>1/2 cup white sugar</li>
+            <li>1/2 cup packed brown sugar</li>
+            <li>8 Granny Smith apples - peeled, cored and sliced</li>
+          </ul>
+        </section>
+
+        <section class="howto">
+          <img
+            id="recipe-info"
+            src="/apple-pie/images/cooking-info.png"
```
Comment (trebeljahr) [inline apple-pie/index.html:59]: depending on which part of a page (which URL you are on) the / before file srcs might not be what you want. 
have a look at this: 
https://webmasters.stackexchange.com/questions/56840/what-is-the-purpose-of-leading-slash-in-html-urls
tldr: it might break when you are nested in sites and your image src is not nested :+1:

File: apple-pie/styles/style.css [hunk 4]
```diff
@@ -0,0 +1,147 @@
+body,
+html {
+  margin: 0;
+  padding: 0;
+  /* font-family: "Roboto", sans-serif; */
+  font-family: "Open Sans", sans-serif;
```
Comment (trebeljahr) [inline apple-pie/styles/style.css:6]: use the family that you imported from google :)

**Review by trebeljahr — COMMENTED** [review]
really good job :) outstanding work. keep it up like this :+1:

File: apple-pie/styles/style.css (hunk 4 above)
Comment (hvkst) [reply apple-pie/styles/style.css:6]: Yeah, I wasn´t really happy with Roboto and in the French Soup css was Open Sans, so I used that in both :)
